### ISSUE TITLE
Make "Expert mode required" message consistent

### ISSUE THREAD

**Issue by matevz** [Zondax/ledger-oasis#192]
When signing transaction which requires expert mode, the current UI on my Nano S is slightly broken (see attachment).

If possible, use "This app is not genuine" interface with an exclamation mark on the left

```
Signing rejected
Expert mode required
```

No left-right screens needed, double-click should hide the message.

![IMG_20230213_115114297](https://user-images.githubusercontent.com/83672/218442556-b4e8d666-d878-4444-bdbf-529acd2716db.jpg)

:link: [zboto Link](https://app.clickup.com/t/8677d8bmn)

**Comment by chcmedeiros** [comment]
Hey! I already fixed the message in the latest push to be:
```
Signing rejected
Expert mode required
```
Regarding the "This app is not genuine" ui, we can add on the app opening a warning to the user saying the app as not been reviewed by Ledger, which goes away with a double click. When the app is published by Ledger, this warning won't appear.

**Comment by matevz** [comment]
I checked the new version and the typo was fixed, thanks.

My suggestion was that the UI for errors/warnings should be made consistent across the app. A small checkmark on the right with the right-click to continue is currently not intuitive. Either there should be a separate "Cancel" screen with double-click to hide it or a single error screen with a triangle ! icon on the left and double-click hides it.
